core/consensus/aura/primitives/src/lib.rs

```diff
@@ -18,8 +18,13 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+use parity_codec::{Encode, Decode};
 use substrate_client::decl_runtime_apis;
 use runtime_primitives::ConsensusEngineId;
+use rstd::vec::Vec;
+
+#[derive(Eq, PartialEq, Clone, Copy, Encode, Decode, Debug)]
+pub struct AuraEvidence {}
```

Comment:
needs docs. Also, favor `pub struct AuraEvidence;` (braced empty structs should be used in macros only).